doc/examples/color_exposure/plot_ihc_color_separation.py

```diff
@@ -60,22 +61,23 @@
 
 
 ######################################################################
-# Now we can easily manipulate the hematoxylin and DAB "channels":
+# Now we can easily manipulate the hematoxylin and DAB "channels:"
```

Comment:
Is this change intentional?

Reply:
Yes, I try to follow [The Canadian Style](https://www.btb.termiumplus.gc.ca/tcdnstyl-chap?lang=eng&lettr=chapsect7&info0=7.18) but maybe I didn't get this right?

Reply:
As channels is not used between quotes [here](https://github.com/scikit-image/scikit-image/pull/5231/files/32ec5fcab4c5b296ceeb95a2138555adcad2ffe9#diff-3b977591f3df3413ae3f72b36779dfa715cd5a12058383a9a6e86f23c836364eR68), I think that we can remove them to avoid confusions
```suggestion
# Now we can easily manipulate the hematoxylin and DAB channels:
```